RELEASES.md

```diff
@@ -80,3 +80,47 @@ popd
 - [Packages](https://github.com/orgs/ROCm/packages?repo_name=TheRock): We currently publish docker images for LLVM targets we support (as well as a container for our build machines)
 
 - [Per-commit CI builds](https://github.com/ROCm/TheRock/actions/workflows/ci.yml?query=branch%3Amain+is%3Asuccess): Each of our latest passing CI builds has its own artifacts you can leverage. This is the latest and greatest! We will eventually support a nightly release that is at a higher quality bar than CI. Note a quick recipe for getting all of these from the s3 bucket is to use this quick command `aws s3 cp s3://therock-artifacts . --recursive --exclude "*" --include "${RUN_ID}-${OPERATING_SYSTEM}/*.tar.xz" --no-sign-request` where ${RUN_ID} is the runner id you selected (see the URL). Check the [AWS docs](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) to get the aws cli.
+
+## Using our Python packages
+
+The Python packages contain pre-build libraries and tools but do not provide Python APIs for import or usage. Rather, pip is leveraged as a convient way to install the pre-build libraries and tools. For now, Python packages are only supported on Linux.
```

Comment:
grammar

```suggestion
The Python packages contain pre-built libraries and tools but do not provide Python APIs for import or usage. Rather, pip is leveraged as a convient way to install the pre-build libraries and tools. For now, Python packages are only supported on Linux.
```

Another thing we can do for "For now, Python packages are only supported on Linux." is to provide a support table like https://github.com/ROCm/TheRock/blob/0f97bd525e6d1d0f71c1ac70149ecdd9e9fc4125/README.md?plain=1#L21-L24

When reading paragraphs of text, it is easy to miss one sentence in the middle or even at the end. Tables and other visual elements can break up the flow and make it easier to see information at a glance.

Reply:
SGTM, I'll add a table.

Reply:
Added a table.